netmosquito.py

```diff
@@ -0,0 +1,68 @@
+import socket, sys
+
+buffer_size = 1				#How much data would you like to receive at once (bytes)
+timeout_time= 2				#Timeout to not keep holding forever in case of incomplete packet
+
+if len(sys.argv)==3:
+	host=sys.argv[1] 		#Host IP goes here
+	port=int(sys.argv[2]) 	#Port Goes Here
+
+else:
+	print(('Error while running, please execute proggram as following:\npython netmosquito.py <host> <port>'))
+	exit()
+
+s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+s.connect((host,port))
+print(f'Connection established with {host}:{str(port)}\n')
+s.settimeout(timeout_time)	#Sets the timeout time
+
+def receive_data(s):
+	#This functions recieves data in n bytes chunks until '\n' is received or timeout ocurrs
```

Comment:
Commentary in function without [doc string](https://www.askpython.com/python/python-comments)

Reply:
Unaligned commentary